mscore/prefsdialog.cpp

```diff
@@ -342,132 +580,17 @@ void PreferenceDialog::updateValues(bool useDefaultValues)
-      useJackAudio->setChecked(preferences.getBool(PREF_IO_JACK_USEJACKAUDIO));
-      portaudioDriver->setChecked(preferences.getBool(PREF_IO_PORTAUDIO_USEPORTAUDIO));
-      pulseaudioDriver->setChecked(preferences.getBool(PREF_IO_PULSEAUDIO_USEPULSEAUDIO));
-      useJackMidi->setChecked(preferences.getBool(PREF_IO_JACK_USEJACKMIDI));
-      useJackTransport->setChecked(preferences.getBool(PREF_IO_JACK_USEJACKTRANSPORT));
-      becomeTimebaseMaster->setChecked(preferences.getBool(PREF_IO_JACK_TIMEBASEMASTER));
-
-      alsaDevice->setText(preferences.getString(PREF_IO_ALSA_DEVICE));
-
-      int index = alsaSampleRate->findData(preferences.getInt(PREF_IO_ALSA_SAMPLERATE));
-      alsaSampleRate->setCurrentIndex(index);
-
-      index = alsaPeriodSize->findData(preferences.getInt(PREF_IO_ALSA_PERIODSIZE));
-      alsaPeriodSize->setCurrentIndex(index);
-
-      alsaFragments->setValue(preferences.getInt(PREF_IO_ALSA_FRAGMENTS));
-      drawAntialiased->setChecked(preferences.getBool(PREF_UI_CANVAS_MISC_ANTIALIASEDDRAWING));
-      limitScrollArea->setChecked(preferences.getBool(PREF_UI_CANVAS_SCROLL_LIMITSCROLLAREA));
-      switch (preferences.sessionStart()) {
-            case SessionStart::EMPTY:  emptySession->setChecked(true); break;
-            case SessionStart::LAST:   lastSession->setChecked(true); break;
-            case SessionStart::NEW:    newSession->setChecked(true); break;
-            case SessionStart::SCORE:  scoreSession->setChecked(true); break;
-            }
-      sessionScore->setText(preferences.getString(PREF_APP_STARTUP_STARTSCORE));
-      expandRepeats->setChecked(preferences.getBool(PREF_IO_MIDI_EXPANDREPEATS));
-      normalize->setChecked(preferences.getBool(PREF_EXPORT_AUDIO_NORMALIZE));
-      exportRPNs->setChecked(preferences.getBool(PREF_IO_MIDI_EXPORTRPNS));
-      instrumentList1->setText(preferences.getString(PREF_APP_PATHS_INSTRUMENTLIST1));
-      instrumentList2->setText(preferences.getString(PREF_APP_PATHS_INSTRUMENTLIST2));
 
-      importLayout->setChecked(preferences.getBool(PREF_IMPORT_MUSICXML_IMPORTLAYOUT));
-      importBreaks->setChecked(preferences.getBool(PREF_IMPORT_MUSICXML_IMPORTBREAKS));
+      jackDriver->setChecked(preferences.getBool(PREF_IO_JACK_USEJACKAUDIO) || preferences.getBool(PREF_IO_JACK_USEJACKMIDI));
 
 #ifdef AVSOMR
-      useLocalAvsOmr->setChecked(preferences.getBool(PREF_IMPORT_AVSOMR_USELOCAL));
-      Avs::AvsOmrLocal::instance()->isInstalledAsync([this](bool isInstalled) {
-            QString text = QObject::tr("Use local OMR engine");
-            if (isInstalled)
-                  text += " (" + QObject::tr("Installed") + ")";
-            else
-                  text += " (" + QObject::tr("Not installed, needs internet connection for installing") + ")";
-
-            useLocalAvsOmr->setText(text);
-            });
+      //added as a BoolPreferenceItem
```

Comment:
why not get rid of this comment and only use
```
#ifndef AVSOMR
    groupBox_omr->setVisible(false);
#endif
```
this comment doesn't seem very useful to me.

Reply:
You are right, it's useless. I will delete that.